### PR TITLE
scrolling overflowing blocks into viewport for inline editing

### LINKED ISSUE #908: Scroll inline editor to prevent cropping
screenshot 2015-02-04 11 43 35

Repro: Tap edit on a block near to the top or bottom of the viewport.
Actual: There is some cropping of the inline editor.
Expected: Inline editor adjusts itself to prevent overflow.

### PR DESCRIPTION
**Pull request by gvn**
Closes #908 

R? @k88hudson

### CODE REVIEW

**Comment by k88hudson** [comment]
Having trouble reviewing this since https://github.com/mozilla/webmaker-app/issues/917 is causing issues on 4.4 -- mind taking a look at that one first?

**Comment by gvn** [comment]
Can you give it a shot now @k88hudson ?

**Comment by k88hudson** [comment]
yep will do right after lunch

**Comment by k88hudson** [comment]
R+. This was actually so seamless i really had to think about what was actually different heh.

File: components/inlineEditor/index.js [hunk 1]
```diff
@@ -18,6 +18,55 @@ module.exports = {
...
             this.$dispatch('inlineEditorStarted', {index: this.$index});
             this.$el.classList.add('active');
+            this.centerEditor();
+        },
+        centerEditor: function () {
+            // Elements
+            var elEditor = this.$el;
+            var elContainer = elEditor.offsetParent;
+            var elControlPanel = elEditor.querySelector('.control-panel');
+            var elNavBar = document.querySelector('#navigationBar');
```
Comment (k88hudson) [inline components/inlineEditor/index.js]: this is a bit fragile

Comment (k88hudson) [inline components/inlineEditor/index.js]: I would maybe store these as constants up top or something maybe

**Comment by k88hudson** [comment]
1 nit

**Pull request merged**